packages/carrier-mobile-ionic/src/components/orders-list/orders-list.component.html

```diff
@@ -0,0 +1,44 @@
+<ion-header>
+	<ion-toolbar color="primary">
+		<ion-title>Select Order for Delivery ...</ion-title>
+	</ion-toolbar>
+</ion-header>
+
+<ion-content fullscreen color="light">
+	<ion-card button *ngFor="let order of filteredList">
+		<ion-card-content (click)="selectNewOrder(order.id)">
+			<ion-item>
+				<ion-avatar slot="start">
+					<img [src]="order.warehouse.logo" />
+				</ion-avatar>
+				<ion-label>
+					<p>name: {{ order.warehouse.name }}</p>
```

Comment:
@valiopld why "name", but "Phone", where is consistency (even if it's ugly, things should be consistently ugly)